geoportailv3/static/js/interrogation/interrogationdirective.js

```diff
@@ -0,0 +1,231 @@
+goog.provide('app.interrogationDirective');
+
+
+/**
+ * @param {string} appInterrogationTemplateUrl URL to directive template.
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+app.interrogationDirective = function(appInterrogationTemplateUrl) {
```

Comment:
After renaming the directory, I had to make a cleanall to have it working in production mode.